src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,121 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private int capacity = DEFAULT_CAPACITY;
+    private Node<K, V>[] table = new Node[capacity];
+    private int size = 0;
+    private int threshold = 0;
```

Comment:
The threshold field is never initialized, but it's used in the put() method. It seems that the threshold should be initialized as `capacity * LOAD_FACTOR` in the constructor. Also, consider making the threshold field `final` if it's not supposed to change during the lifetime of a MyHashMap instance.